api/infrastructure/parallelcluster-api.yaml

```diff
@@ -142,6 +149,412 @@ Resources:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: S3ResourcesBucket
+        - Action:
+          - s3:Get*
+          - s3:List*
+          Resource: !Sub
+          - arn:${AWS::Partition}:s3:::${Region}-aws-parallelcluster*
+          - { Region: !If [IsMultiRegion, '*', !Ref AWS::Region] }
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: S3ParallelClusterReadOnly
+        - Action:
+          - fsx:*
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: FSx
+        - Action:
+          - ecr:*
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: ECR
+        - Action:
+          - batch:*
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: Batch
+        - Action:
+          - events:*
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Resource: '*'
+          Sid: AmazonCloudWatchEvents
+        - Action:
+          - ssm:GetParametersByPath
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: SSMDescribe
+        - Action:
+          - ecs:DescribeContainerInstances
+          - ecs:ListContainerInstances
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: ECS
+        - Action:
+          - elasticfilesystem:CreateFileSystem
+          - elasticfilesystem:CreateMountTarget
+          - elasticfilesystem:DeleteFileSystem
+          - elasticfilesystem:DeleteMountTarget
+          - elasticfilesystem:DescribeFileSystems
+          - elasticfilesystem:DescribeMountTargets
+          - elasticfilesystem:DescribeMountTargetSecurityGroups
+          - ec2:DescribeNetworkInterfaceAttribute
```

Comment:
shall we move this to the EC2 section?
Also we can probably due elasticfilesystem:* as done for fsx

Reply:
Sure, I had left it here because it was where it was placed by the [SGE/Slurm/Torque policy](https://docs.aws.amazon.com/parallelcluster/latest/ug/iam.html#parallelclusteruserpolicy)